store.go

```diff
@@ -3305,6 +3361,9 @@ func copyStringInterfaceMap(m map[string]interface{}) map[string]interface{} {
 // defaultConfigFile path to the system wide storage.conf file
 const defaultConfigFile = "/etc/containers/storage.conf"
 
+const AutoUserNsMinSize = 1024
```

Comment:
The docs state `4096`. Which value is the right one?

Reply:
updated the docs to the correct value